lisa/trace.py

```diff
@@ -143,6 +143,7 @@ def cpus_count(self):
         # If we don't know the number of CPUs, check the trace for the
         # highest-numbered CPU that traced an event.
         except KeyError:
+            self.get_logger().info("Estimating CPU count from trace")
```

Comment:
Can we log that after the estimation and show the estimated number as well ? That can speed up debugging if the log already contains the info straight away